test/lint-algorithm-line-endings.js

```diff
@@ -93,6 +108,9 @@ describe('linting algorithms', function () {
           1. Other.
           1. Other:
             1. Substep.
+          1. Let _constructorText_ be the source text
+          <pre><code class="javascript">constructor() {}</code></pre>
+          1. Set _constructor_ to ParseText(_constructorText_, |MethodDefinition[~Yield, ~Await]|).
```

Comment:
Was this line needed?

Reply:
It isn't strictly needed. I just wanted to confirm that the PR this change is intended to allow will in fact be allowed by it.